docs/pages/includes/enterprise/docker-images.mdx

```diff
@@ -4,6 +4,10 @@ automatically rebuilt every night.
 Nightly images point to the latest version of Teleport Enterprise from the three most recent release branches.
 They are stable, and we recommend their use to easily keep your Teleport Enterprise installation up to date.
 
+These images are hosted on Amazon ECR Public. All tags under `public.ecr.aws/gravitational/teleport-ent` [are Teleport Enterprise images](https://gallery.ecr.aws/gravitational/teleport-ent).
+Versions without a architecture postfix type are `amd64` architecute. (public.ecr.aws/gravitational/teleport:(=teleport.major_version=)). All architecture versions are available 
+with the postfix type (ex: `public.ecr.aws/gravitational/teleport:(=teleport.major_version=)-arm64`).
```

Comment:
I think we can make "All architecture versions are available with the postfix type" clearer. Is this saying that you can pull an image with the `amd64`, `arm`, or `arm64` architecture by specifying it as a suffix?

Reply:
Yes, that's correct.  I can change to suffix instead of postfix.